<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWidget</class>
 <widget class="QWidget" name="SelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>38</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>514</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame{	
	border-radius: 2px;
	background-color: rgba(0, 0, 0, 214);
}
QPushButton#pushButtonSmall {
    background-color: rgba(176, 176, 176, 1);
    border-radius: 3px;
}
QPushButton#pushButtonSmall:hover {
}
QPushButton#pushButtonSmall:checked {
    background-color:rgba(0, 89, 255, 1);
}

QPushButton#pushButtonMiddle{
    background-color: rgba(176, 176, 176, 1);
    border-radius: 5px;
}
QPushButton#pushButtonMiddle:hover {
}
QPushButton#pushButtonMiddle:checked {
    background-color:rgba(0, 89, 255, 1);
}

QPushButton#pushButtonLarge {
    background-color: rgba(176, 176, 176, 1);
    border-radius: 7px;
}
QPushButton#pushButtonLarge:hover {
}
QPushButton#pushButtonLarge:checked {
    background-color:rgba(0, 89, 255, 1);
}

QPushButton#pushButtonRed {
    background-color: rgba(250, 0, 0, 1);
    border-radius: 1px;
}
QPushButton#pushButtonRed:hover {
    border-image:url(:/image/pc_screenshot_color_hover.png);
}
QPushButton#pushButtonRed:checked {
    border-image:url(:/image/pc_screenshot_color_selected.png);
}

QPushButton#pushButtonYellow {
    background-color: rgba(255, 204, 0, 1);
    border-radius: 1px;
}
QPushButton#pushButtonYellow:hover {
    border-image:url(:/image/pc_screenshot_color_hover.png);
}
QPushButton#pushButtonYellow:checked {
    border-image:url(:/image/pc_screenshot_color_selected.png);
}

QPushButton#pushButtonBlue {
    background-color: rgba(69, 134, 255, 1);
    border-radius: 1px;
}
QPushButton#pushButtonBlue:hover {
    border-image:url(:/image/pc_screenshot_color_hover.png);
}
QPushButton#pushButtonBlue:checked {
    border-image:url(:/image/pc_screenshot_color_selected.png);
}

QPushButton#pushButtonGreen {
    background-color: rgba(116, 224, 0, 1);
    border-radius: 1px;
}
QPushButton#pushButtonGreen:hover {
    border-image:url(:/image/pc_screenshot_color_hover.png);
}
QPushButton#pushButtonGreen:checked {
    border-image:url(:/image/pc_screenshot_color_selected.png);
}

QPushButton#pushButtonGray {
    background-color: rgba(74, 74, 74, 1);
    border-radius: 1px;
}
QPushButton#pushButtonGray:hover {
    border-image:url(:/image/pc_screenshot_color_hover.png);
}
QPushButton#pushButtonGray:checked {
    border-image:url(:/image/pc_screenshot_color_selected.png);
}

QPushButton#pushButtonWhite{
    background-color: rgba(255, 255, 255, 1);
    border-radius: 1px;
}
QPushButton#pushButtonWhite:hover {
    border-image:url(:/image/pc_screenshot_color_hover_white.png);
}
QPushButton#pushButtonWhite:checked {
    border-image:url(:/image/pc_screenshot_color_selected_white.png);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>245</width>
     <height>38</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>1</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>18</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1</width>
      <height>18</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.30);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSmall">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>16</y>
      <width>6</width>
      <height>6</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>6</width>
      <height>6</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>6</width>
      <height>6</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMiddle">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>14</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLarge">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>12</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBlue">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonYellow">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRed">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGreen">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGray">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWhite">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
